lib/services/subscription_handler.dart

```diff
@@ -36,6 +41,39 @@ Future<void> subscribeForArea({
     context: context,
     text: localizations.loading_screen_loading,
   );
+
+  LoadingScreen.instance().show(
+    context: context,
+    text: localizations.loading_screen_wait_for_push_to_complete,
+  );
+
+  var userPreferences = ref.read(userPreferencesProvider);
+  debugPrint(
+    "wait for registration state=${userPreferences.unifiedPushRegistered}",
+  );
+  // wait for the registration to finish. This shouldn't be necessary as we
```

Comment:
Since it is necessary as I explain in my other comment, this comment is obsolete.